### PR TITLE
Move dialogue screen into survey edit page

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review

File: go/apps/surveys/tests/test_views.py [hunk 1]
```diff
@@ -281,60 +241,6 @@ def test_export_messages(self):
...
-        self.assertEqual(response1.context['delay'], 1.1 * 100)
-        # Second time it should still render the messages
-        self.assertTrue('generic/includes/message-list.html' in
-                            [t.name for t in response2.templates])
-        self.assertEqual(response1.context['token'], fake_client.token)
-        # Second time we should list the matching messages
-        self.assertEqual(response2.context['token'], fake_client.token)
-        self.assertEqual(len(response2.context['message_page'].object_list),
-            10)
-
```
Comment (hodgestar) [inline go/apps/surveys/tests/test_views.py:337]: I think these should hang around to remind us that the new views need to have this.

File: go/wizard/views.py [hunk 2]
```diff
@@ -95,20 +95,6 @@ def edit(request, conversation_key):
...
-        'conversation_key': conversation_key,
-        'conversation': conversation,
-        'model_data': model_data
-    })
-
-
-@login_required
 def edit_bulk_message(request, conversation_key):
     """The simpler of the two messages."""
     conversation = conversation_or_404(request.user_api, conversation_key)
```
Comment (hodgestar) [inline go/wizard/views.py:100]: I think all of these edit views can go (can be in a separate PR though).

**Comment by hodgestar** [comment]
Looks good -- just left two small comments.

**Comment by hodgestar** [comment]
:+1: as soon as Travis finishes.
